packages/pipeline-editor/src/utils.ts

```diff
@@ -33,27 +32,34 @@ export default class Utils {
    */
   static generateNotebookPipeline(
     filename: string,
-    options: ISubmitNotebookOptions
+    runtime_config: string,
+    framework: string,
+    dependencies: string[],
+    envObject: { [key: string]: string }
   ): any {
     const template = JSON.parse(JSON.stringify(pipeline_template));
     const generated_uuid: string = Utils.getUUID();
 
     const artifactFileName = filename.replace(/^.*[\\/]/, '');
     const artifactName = artifactFileName.replace(/\.[^/.]+$/, '');
 
+    const envArray = Object.entries(envObject).map(
+      ([key, val]) => `${key}=${val}`
+    );
+
     template.id = generated_uuid;
     template.primary_pipeline = generated_uuid;
     template.pipelines[0].id = generated_uuid;
 
     template.pipelines[0].nodes[0].id = generated_uuid;
     template.pipelines[0].nodes[0].app_data.filename = filename;
-    template.pipelines[0].nodes[0].app_data.runtime_image = options.framework;
-    template.pipelines[0].nodes[0].app_data.env_vars = options.env;
-    template.pipelines[0].nodes[0].app_data.dependencies = options.dependencies;
+    template.pipelines[0].nodes[0].app_data.runtime_image = framework;
+    template.pipelines[0].nodes[0].app_data.env_vars = envArray;
```

Comment:
we recently had some updates on trying to use same/similar names for properties to make reading much simpler,  could we use something like `envVars` or `envVarsArray` here?